test/run_test

```diff
@@ -132,7 +135,7 @@ function create_container() {
   local cargs=${DOCKER_ARGS:-}
   cidfile="$CIDFILE_DIR/$name"
   # create container with a cidfile in a directory for cleanup
-  docker run $cargs --cidfile $cidfile -d $IMAGE_NAME "$@"
+  docker run $cargs --cidfile $cidfile -d "$@" $IMAGE_NAME
```

Comment:
This doesn't seem right.  `$@` brings the actual command to be run inside the container.